sources/core/objects/turfs/MetalWall.cpp

```diff
@@ -45,7 +111,27 @@ ReinforcedWall::ReinforcedWall()
     SetVisibleLevel(visible_level::HIGH_TURF);
 
     SetSprite("icons/walls.dmi");
-    SetState("r_wall");
 
     SetName("Reinforced wall");
+
+    default_state_="rwall";
+    current_state_=0;
+
+    SetState("rwall0");
+}
+
+void ReinforcedWall::AfterWorldCreation()
+{
+    UpdateState(Dir::ALL);
+}
+
+void ReinforcedWall::Delete()
```

Comment:
I think, this one can be just deleted - nothing should change because of that function.